README.md

```diff
@@ -108,6 +108,15 @@ Type: `String`
 Default: `'html'`
 Available: `['pretty', 'progress', 'summary', 'html']`
 
+#### options.formats
+Supports multiple formatter.
+Type: `Array`
+Available: `['pretty', 'progress', 'summary', 'html']`
+
+e.g. `formats: ['html', 'pretty']`
+
+Note: `html` formatter will provide Json as well as `html` report. Multiple formatter is supported for cucumber v@0.8.0 or higher
+
```

Comment:
Should we add a `peerDependency` in package.json to make sure that `cucumber@^0.8.0`? This way we make sure that users do have the correct dependencies prior to using this feature.